core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -5006,6 +5006,17 @@ secret_configs:
   - cluster: app.ci
     name: ci-image-mirror
     namespace: ci
+- from:
+    config.json:
```

Comment:
```suggestion
    .dockerconfigjson:
```

Reply:
The deployment has to recognize the file or rename via its configuration.

Reply:
the name should be `config.json`. Is this hardcoded in the secret generator?